增加动画？

Lex-monkey opened the pull request:
new file:   src/components/AnimatedCard.module.css
	modified:   src/components/HomepageFeatures/index.js
	modified:   src/css/custom.css
	modified:   src/pages/index.js
	new file:   static/js/animation.js
	modified:   static/js/script.js

Lex-monkey committed:
new file:   src/components/AnimatedCard.js
	new file:   src/components/AnimatedCard.module.css
	modified:   src/components/HomepageFeatures/index.js
	modified:   src/css/custom.css
	modified:   src/pages/index.js
	new file:   static/js/animation.js
	modified:   static/js/script.js

Lex-monkey committed:
new file:   src/components/FloatingElements.js
	new file:   src/components/FloatingElements.module.css
	modified:   src/components/HomepageFeatures/index.js
	new file:   src/components/PageLoader.js
	new file:   src/components/PageLoader.module.css
	modified:   src/pages/index.js
	modified:   src/pages/index.module.css

Lex-monkey committed:
modified:   src/components/HomepageFeatures/index.js
	modified:   src/components/HomepageFeatures/styles.module.css
	modified:   src/css/custom.css
	modified:   src/pages/index.js
	modified:   src/pages/index.module.css
	modified:   static/js/animation.js

Lex-monkey committed:
modified:   src/css/custom.css
	modified:   src/pages/index.module.css
	modified:   static/js/animation.js

Lex-monkey committed:
Merge branch 'main' into lexdev

Moralts commented on static/js/animation.js:
```diff
@@ -0,0 +1,97 @@
+  fadeElements.forEach((el, index) => {
+    el.style.opacity = '0';
+    el.style.transform = 'translateY(20px)';
+    el.style.transition = 'opacity 0.8s ease-out, transform 0.8s ease-out';
```
可以缩短到 0.5s 或者更快，防止视觉上的加载缓慢

Moralts commented on static/js/script.js:
请不要把原来的脚本顶替

Moralts replied:
已经修补

Moralts commented on src/components/PageLoader.js:
没有必要加入加载动画

Moralts commented on src/components/HomepageFeatures/index.js:
主页貌似重叠部分较多，保持原样即可

Lex-monkey committed:
modified:   src/css/custom.css

Lex-monkey committed:
modified:   src/css/custom.css
	modified:   src/pages/index.module.css
	new file:   static/img/indexpho.jpg
	modified:   static/js/animation.js

Moralts commented on src/components/AnimatedCard.js:
未发现卡片动画与原版的有任何区别

Moralts commented on src/css/custom.css:
```diff
@@ -39,25 +38,191 @@
+.footer__link,
+.footer__title {
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 0.95rem;
```
字体保持原样，有点过小

Moralts commented on static/img/indexpho.jpg:
未找到使用处

pingguomc reviewed (commented):
流畅、简约，感觉还不错。有技术力

个人觉得也许从左侧划入不如淡入好。

Commit:
扩展为四边阴影

Commit:
去除虚化和边框，平均间隔

Commit:
去除加载动画

Commit:
去除加载动画

Commit:
更改css样式

Commit:
更改主页样式

Commit:
更新css样式

Commit:
合并 pangu 脚本加载

Moralts reviewed (commented):
二改完毕，确认合并拉取请求，感谢你的贡献🎉
